app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -941,13 +958,13 @@ export class AppCommandParser {
         }
 
         context.channel_id = channel.id;
-        context.team_id = channel.team_id || getCurrentTeamId(this.store.getState());
+        context.team_id = this.teamID || channel.team_id || getCurrentTeamId(this.store.getState());
```

Comment:
When would `this.teamID` be more useful than `channel.team_id` and vise versa?

Reply:
This team ID is more useful than channel.teamID on dms, since dms dont have team id.
Also, on mobile the current team id that comes from the store might not be the same as the channel team id.